src/clj/puppetlabs/puppetserver/cli/subcommand.clj

```diff
@@ -52,3 +52,11 @@
       (throw+))
     (finally
       (shutdown-agents))))
+
+(defn jruby-run
```

Comment:
Agreed on this - if we get other consumers of jruby-utils then I think it'll be worth revisiting this, but for the time being it doesn't seem worth the effort.